easybuild/easyconfigs/g/GEMMA/GEMMA-0.98.4-foss-2020a.eb

```diff
@@ -0,0 +1,38 @@
+# This file is an EasyBuild reciPY as per https://easybuilders.github.io/easybuild/
+# Author: Pablo Escobar Lopez
+# sciCORE - University of Basel
+# SIB Swiss Institute of Bioinformatics
+
+easyblock = 'MakeCp'
+
+name = 'GEMMA'
+version = '0.98.4'
+
+homepage = 'https://github.com/genetics-statistics/GEMMA'
+description = "Genome-wide Efficient Mixed Model Association"
+
+toolchain = {'name': 'foss', 'version': '2020a'}
+
+source_urls = ['https://github.com/genetics-statistics/GEMMA/archive/refs/tags']
+sources = ['v%(version)s.tar.gz']
+checksums = ['4f57a045d3289afaf31f818bf411ac46c5ee6f78ff8c9c4117963ca54e0bb9f0']
+
+builddependencies = [
+    ('Eigen', '3.3.7', '', True),
+]
+
+dependencies = [
+    ('GSL', '2.6'),
+    ('zlib', '1.2.11')
+]
+
+files_to_copy = ["bin", "doc", "example", "LICENSE", "README.md", "RELEASE-NOTES.md", "scripts", "VERSION"]
+
+sanity_check_commands = ["cd %(builddir)s/%(name)s-%(version)s/ && ./run_tests.sh"]
+
+sanity_check_paths = {
+    'files': ["bin/gemma"],
+    'dirs': []
+}
+
```

Comment:
Since Kenneth typically mentions it to me, perhaps add a sanity_check_command here `gemma -h`

Reply:
I totally missed the run_test.sh that already exists, doh!